app/components/CapitalProjectDetailPanel/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { Flex } from "@nycplanning/streetscape";
+import {
+  MobilePanelSizeControl,
+  MobilePanelSizeControlProps,
+} from "../MobilePanelSizeControl";
+import {
+  CapitalProjectDetail,
+  CapitalProjectDetailProps,
+} from "./CapitalProjectDetail";
+export { CapitalProjectDetail } from "./CapitalProjectDetail";
+
+export interface CapitalProjectDetailPanelProps
+  extends MobilePanelSizeControlProps,
+    CapitalProjectDetailProps {}
+
+export function CapitalProjectDetailPanel(
+  props: CapitalProjectDetailPanelProps,
+) {
+  return (
+    <Flex
+      borderTopRadius={"base"}
+      borderBottomRadius={{ base: "0", lg: "base" }}
+      background={"white"}
+      direction={"column"}
+      width={{ base: "full", lg: "21.25rem" }}
+      maxW={{ lg: "unset" }}
+      height={{ base: props.isExpanded ? "90vh" : "auto", lg: "auto" }}
```

Comment:
I wonder if we need to specify the non expanded height. From my local machine, a single project view lower on the screen than what's in the designs (which is closer to about half the viewport).

Reply:
Yeah, these aren't great values. But, things will shift when we add the commitments timeline. I think it makes sense to wait until we add the commitments timeline to reconsider the values.